openreview/conference/templates/recruitReviewersProcess.py

```diff
@@ -0,0 +1,56 @@
+def process(client, note, invitation):
+    from Crypto.Hash import HMAC, SHA256
+    import urllib.parse
+    SHORT_PHRASE = ''
+    CONFERENCE_NAME = ''
+    REVIEWER_NAME = ''
+    REVIEWERS_INVITED_ID = ''
+    REVIEWERS_ACCEPTED_ID = ''
+    REVIEWERS_DECLINED_ID = ''
+    AREA_CHAIR_NAME = ''
+    AREA_CHAIRS_ACCEPTED_ID = ''
+    HASH_SEED = ''
+    REDUCED_LOAD_INVITATION_NAME = ''
+
+    user = urllib.parse.unquote(note.content['user'])
+
+    hashkey = HMAC.new(HASH_SEED.encode(), digestmod=SHA256).update(user.encode()).hexdigest()
+    
+    invited_group = client.get_group(REVIEWERS_INVITED_ID)
+    if (hashkey == note.content['key'] and user in invited_group.members):
+        if (note.content['response'] == 'Yes'):
+            if (AREA_CHAIRS_ACCEPTED_ID and user in client.get_group(AREA_CHAIRS_ACCEPTED_ID).members):
```

Comment:
In order to check any other user email/tilde id, we can do the following:

```
client.get_groups(regex=AREA_CHAIRS_ACCEPTED_ID, member=user)
```

If the result is not empty then the user accepted to be an area chair.